src/shared/cache/PmidCache.ts

```diff
@@ -1,53 +1,39 @@
-import {default as SimpleCache, ICache} from "shared/lib/SimpleCache";
+import LazyMobXCache from "../lib/LazyMobXCache";
+import {AugmentedData} from "../lib/LazyMobXCache";
 
-export default class PmidCache extends SimpleCache<any, number[]>
+async function fetch(pmids: number[])
 {
-    constructor()
-    {
-        super();
-    }
-
-    protected async fetch(pmids: number[])
-    {
-        const cache: ICache<any> = {};
-
-        try {
-            const pmidData:any = await new Promise((resolve, reject) => {
-                // TODO better to separate this call to a configurable client
-                $.post(
-                    'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&retmode=json',
-                    {id: pmids.join(',')}
-                ).then((data) => {
-                    resolve(data);
-                }).fail((err) => {
-                    reject(err);
-                });
+        const pmidData:any = await new Promise((resolve, reject) => {
```

Comment:
lets always use superagent library b/c we get benefit of error handling and Sentry tracking out of box